sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -531,7 +531,7 @@ export class EventHubReceiver extends LinkEntity {
         delayInSeconds: 15,
         operation: () => this.initialize(initOptions),
         operationType: RetryOperationType.receiverLink,
-        maxRetries: Constants.defaultMaxRetriesForConnection
+        maxRetries: Constants.defaultMaxRetriesForConnection,
```

Comment:
Super nit: looks like this change wasn't needed, does running `npm run format` remove the comma?

Reply:
That's odd, it wasn't going away on file save. After merge from master it disappears on file save itself..